modules/xmpp/ChatRoom.js

```diff
@@ -1861,6 +1861,18 @@ export default class ChatRoom extends Listenable {
 
         return Promise.allSettled(promises);
     }
+
+    /**
+     * Ends the conference.
```

Comment:
Maybe clarify that this affects all participants, and not just the local one.

Reply:
I will add a note that this will end the conference for all participants. (The end function is in contrast to the leave function.)

Reply:
👍